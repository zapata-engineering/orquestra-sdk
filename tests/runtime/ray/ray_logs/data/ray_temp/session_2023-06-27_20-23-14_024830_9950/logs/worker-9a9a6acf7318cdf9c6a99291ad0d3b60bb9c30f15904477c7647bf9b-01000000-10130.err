:job_id:01000000
:task_name:_workflow_task_executor_remote
2023-06-27 20:24:32,744	INFO task_executor.py:78 -- Task status [RUNNING]	[wf.wf_using_python_imports.2618433@invocation-0-task-task-with-python-imports]
ORQ-MARKER:{"event": "task_start", "timestamp": "2023-06-27T20:24:32.751341+02:00", "wf_run_id": "wf.wf_using_python_imports.2618433", "task_inv_id": "invocation-0-task-task-with-python-imports"}
ORQ-MARKER:{"event": "task_end", "timestamp": "2023-06-27T20:24:33.383024+02:00", "wf_run_id": "wf.wf_using_python_imports.2618433", "task_inv_id": "invocation-0-task-task-with-python-imports"}
