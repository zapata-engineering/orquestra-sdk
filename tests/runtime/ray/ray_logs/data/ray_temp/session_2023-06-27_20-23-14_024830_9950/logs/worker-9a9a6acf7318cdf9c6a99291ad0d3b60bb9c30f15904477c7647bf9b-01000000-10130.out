:job_id:01000000
:task_name:_workflow_task_executor_remote
ORQ-MARKER:{"event": "task_start", "timestamp": "2023-06-27T20:24:32.751341+02:00", "wf_run_id": "wf.wf_using_python_imports.2618433", "task_inv_id": "invocation-0-task-task-with-python-imports"}
ORQ-MARKER:{"event": "task_end", "timestamp": "2023-06-27T20:24:33.383024+02:00", "wf_run_id": "wf.wf_using_python_imports.2618433", "task_inv_id": "invocation-0-task-task-with-python-imports"}
